apps/explorer/src/pages/transaction-result/transaction-summary/ObjectChanges.tsx

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { getGroupByOwner, LocationIdType } from '@mysten/core';
+import { ChevronRight12 } from '@mysten/icons';
+import {
+    type SuiObjectChangeCreated,
+    type SuiObjectChangeMutated,
+    type SuiObjectChangeTransferred,
+} from '@mysten/sui.js';
+import clsx from 'clsx';
+import { useState } from 'react';
+
+import { ExpandableList, ExpandableListItems } from '~/ui/ExpandableList';
+import { AddressLink, ObjectLink } from '~/ui/InternalLink';
+import { Link } from '~/ui/Link';
+import { Text } from '~/ui/Text';
+import {
+    TransactionBlockCard,
+    TransactionBlockCardSection,
+} from '~/ui/TransactionBlockCard';
+
+enum Labels {
+    created = 'Created',
+    mutated = 'Updated',
+    transferred = 'Transfer',
+}
+
+enum ItemLabels {
+    package = 'Package',
+    module = 'Module',
+    function = 'Function',
+}
+
+const expandableListClassName = 'flex flex-col max-h-[300px] gap-3';
```

Comment:
Just define this inline. If you find patterns like this useful, it's time to make a component!